hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/BlockOutputStreamEntryPool.java

```diff
@@ -368,7 +369,10 @@ void hsyncKey(long offset) throws IOException {
       if (keyArgs.getIsMultipartKey()) {
         throw new IOException("Hsync is unsupported for multipart keys.");
       } else {
+        long start = Time.monotonicNowNanos();
```

Comment:
Can we add a helper wrapper static method to run the code that needs to be instrumented and standardize the units reported? OM has similar code added by @duongkame

Reply:
You can add the following to `org.apache.hadoop.util.MetricUtil`.
```
public final class MetricUtil {
....
  public static <E extends IOException> void captureLatencyNs(Consumer<Long> latencySetter, CheckedRunnable<E> block) throws E {
    long start = Time.monotonicNowNanos();
    try {
      block.run();
    } finally {
      latencySetter.accept(Time.monotonicNowNanos() - start);
    }
  }
...
}
```

And use it here
```
MetricUtil.captureLatencyNs(clientMetrics::addDataNodeHsyncLatency, () -> out1.hsync());
```